src/pages/api/manifest.ts

```diff
@@ -17,16 +17,16 @@ export default createManifestHandler({
       id: "app.saleor.klarna",
       name: "Klarna",
       about: packageJson.description,
-      tokenTargetUrl: `${apiBaseURL}/api/register`,
-      appUrl: iframeBaseUrl,
+      tokenTargetUrl: `${apiBaseURL}${env.NEXT_PUBLIC_BASE_PATH}/api/register`,
+      appUrl: `${iframeBaseUrl}${env.NEXT_PUBLIC_BASE_PATH}`,
       permissions: ["HANDLE_PAYMENTS"],
       version: packageJson.version,
       requiredSaleorVersion: ">=3.15.0",
       homepageUrl: "https://docs.saleor.io/docs/3.x/developer/app-store/apps/klarna",
       supportUrl: "https://github.com/saleor/saleor-app-payment-klarna/issues",
       brand: {
         logo: {
-          default: `${apiBaseURL}/logo.png`,
+          default: `${apiBaseURL}${env.NEXT_PUBLIC_BASE_PATH}/logo.png`,
```

Comment:
**question**: Similar to above. Will this not break without the `NEXT_PUBLIC_BASE_PATH`?

Reply:
Yes, is true. The app should not start if those variables are not set.